docs/book/src/cronjob-tutorial/testdata/project/config/default/kustomization.yaml

```diff
@@ -16,7 +16,8 @@ bases:
 - ../crd
 - ../rbac
 - ../manager
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
+# [WEBHOOK] To enable webhook, uncomment all the sections with
+#  [WEBHOOK] prefix including the one in crd/kustomization.yaml
```

Comment:
I would try to avoid having the second line start by `[WEBHOOK]` too. Either mover `with` to the second line or move `[WEBHOOK]` to the first line.